### PR TITLE
fix(cli): unhide storage and storage init commands.

### PR DESCRIPTION
**Pull request by bvtujo**
This PR sets storage commands to visible and fixes a docstring.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -416,8 +398,6 @@ github.com/smartystreets/goconvey v1.6.4/go.mod h1:syvi0/a8iFYH4r/RixwvyeAJjdLS9
 github.com/soheilhy/cmux v0.1.4/go.mod h1:IM3LyeVVIOuxMH7sFAkER9+bJ4dT7Ms6E4xg4kGIyLM=
 github.com/spaolacci/murmur3 v0.0.0-20180118202830-f09979ecbc72/go.mod h1:JwIasOWyU6f++ZhiEuf87xNszmSA2myDM2Kzu9HwQUA=
 github.com/spf13/afero v1.1.2/go.mod h1:j4pytiNVoe2o6bmDsKpLACNPDBIoEAkihy7loJ1B0CQ=
-github.com/spf13/afero v1.3.0 h1:Ysnmjh1Di8EaWaBv40CYR4IdaIsBc5996Gh1oZzCBKk=
-github.com/spf13/afero v1.3.0/go.mod h1:5KUK8ByomD5Ti5Artl0RtHeI5pTF7MIDuXL3yY520V4=
```
Comment (iamhopaul123) [inline go.sum:420]: ugh why these don't go away automatically. Thanks for removing them.

Comment (bvtujo) [reply go.sum:420]: I'm trying to get into the habit of running `go mod tidy` more often.

File: go.mod [hunk 2]
```diff
@@ -24,6 +24,7 @@ require (
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
 	github.com/pelletier/go-toml v1.8.0 // indirect
+	github.com/pkg/errors v0.9.1
```
Comment (efekarakus) [inline go.mod:27]: huh do you know why this was added?

Comment (bvtujo) [reply go.mod:27]: I'm honestly not sure; I ran `go mod tidy` and I don't quite know why this got picked up. 

...

On further investigation it's added in the `task` pkg; looks like it [got auto-added by goimports](https://github.com/aws/copilot-cli/commit/894883886846b3014a2c175842dec6820dc75d6e) in #1145 when just "errors" would have worked.

File: internal/pkg/cli/storage_init.go [hunk 3]
```diff
@@ -567,13 +567,12 @@ func BuildStorageInitCmd() *cobra.Command {
 		GlobalOpts: NewGlobalOpts(),
 	}
 	cmd := &cobra.Command{
-		Hidden: true, //TODO remove when ready for production
-		Use:    "init",
-		Short:  "Creates a new storage table in an environment.",
+		Use:   "init",
+		Short: "Creates a new storage resource in an environment.",
```
Comment (kohidave) [inline internal/pkg/cli/storage_init.go:571]:
```suggestion
		Short: "Creates a new storage config file in the addons directory.",
```

Comment (kohidave) [reply internal/pkg/cli/storage_init.go:571]: What do you think of this? I just want folks to know that it's not going to **create** a resource.

Comment (bvtujo) [reply internal/pkg/cli/storage_init.go:571]: ooh, yes, good point. Way more clear.

File: go.mod (hunk 2 above)
Comment (efekarakus) [reply go.mod]: ooh @Louise15926 can you take a look at this, when working on the "task" pkg  PR can we remove the dependency from the external `errors` and use "import errors" instead?

**Pull request merged**